lint-tasks.yml

```diff
@@ -1,9 +1,10 @@
 version: "3"
 
 vars:
-  G_LINTER_NODEJS_BUILD_DIR: "{{.G_BUILD_DIR}}/linter-nodejs"
-  G_LINTER_NODEJS_BIN_DIR: "{{.G_LINTER_NODEJS_BUILD_DIR}}/bin"
+  G_LATEST_NODEJS_BIN_DIR: "{{.G_LATEST_NODEJS_DIR}}/bin"
   G_LINT_VENV_DIR: "{{.G_BUILD_DIR}}/lint-venv"
+  G_LOG_VIEWER_WEBUI_SRC_DIR: "{{.ROOT_DIR}}/components/log-viewer-webui"
+  G_WEBUI_SRC_DIR: "{{.ROOT_DIR}}/components/webui"
```

Comment:
I had to move those to the global scope or the variables are undefined in task `link:js-fix`.